src/main/java/net/pms/dlna/DLNAResource.java

```diff
@@ -2257,6 +2257,8 @@ private String getDlnaOrgPnFlags(RendererConfiguration mediaRenderer, int locali
 				} else if (mime.equals("video/vnd.dlna.mpeg-tts")) {
 					// patters - on Sony BDP m2ts clips aren't listed without this
 					dlnaOrgPnFlags = "DLNA.ORG_PN=" + getMPEG_TS_EULocalizedValue(localizationValue, media.isHDVideo());
+				} else if (mime.equals(MP4_TYPEMIME)) {
+					dlnaOrgPnFlags = "*";
```

Comment:
@SubJunk If the converted file plays fine in the renderer, it would be difficult to guess what else might be failing. Do you see anything in the logs?